### ISSUE TITLE
Implement AudioWorklet version of Web Csound

### ISSUE THREAD

**Issue by kunstmusik** [csound/csound#864]
An implementation of AudioWorklet is now testable in Chrome Canary:

http://googlechromelabs.github.io/web-audio-samples/audio-worklet/
https://webaudio.github.io/web-audio-api/#AudioWorklet

The Faust project has already done research work into supporting AudioWorklet and have published examples at:

http://faust.grame.fr/modules-worklet/

**Comment by kunstmusik** [comment]
I added a development repo here: 

https://github.com/kunstmusik/csound-audioworklet

to work on writing the AudioWorklet code. The repo has a pre-built Csound binary committed to repo to make development easier.  Once a basic working version is done, plan is to move back in to csound/emscripten folder as an example.

**Comment by kunstmusik** [comment]
This is complete and work is in main repo now.
